<!-- launch_moving_box.launch -->
<launch>
  
    <!-- Launch Gazebo -->
    <!-- <include file="$(find gazebo_ros)/launch/empty_world.launch">
      <arg name="paused" value="false"/>
      <arg name="use_sim_time" value="true"/>
    </include> -->

    <!-- Spawn a cubic obstacle in front of the UR5e robot -->
    <node name="spawn_cubic_obstacle" pkg="gazebo_ros" type="spawn_model"
          args="-model cubic_obstacle -urdf -f $(find fg_gazebo_example)/urdf/cubic_obstacle.urdf -x 0.5 -y 0 -z 0.45" />

    <!-- Start the move_box_node -->
    <!-- <node name="move_box_node" pkg="fg_gazebo_example" type="move_box.py" output="screen"/> -->

</launch>